web_src/js/features/citation.ts

```diff
@@ -41,9 +42,9 @@ export async function initCitationFileCopyContent() {
     citationCopyApa.classList.toggle('primary', !isBibtex);
   };
 
-  document.querySelector('#cite-repo-button')?.addEventListener('click', async (e: MouseEvent & {target: HTMLAnchorElement}) => {
-    const dropdownBtn = e.target.closest('.ui.dropdown.button');
-    dropdownBtn.classList.add('is-loading');
+  document.querySelector('#cite-repo-button')?.addEventListener('click', async () => {
+    modal.modal('show');
+    modal.addClass('is-loading');
```

Comment:
Since it is going to remove jQuery, here you need 2 variables:

```
$modal.modal('show')
elModal.classList.add(...)
```

Reply:
Done in 4373ad1a623e9756507cd6356edf388017f3b239